app/models/schema_contract/initiator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module SchemaContract
+  class Initiator
```

Comment:
I'm open to feedback on what to name this class. I initially called it Runner but was concerned that would confuse it with rails runners, which this is not intended to be. I also considered Creator but it also queues up the job, so it does more than just create the instance. I also considered Initializer, but that seemed bad because of the initialize method. So basically rails uses all the good descriptors.

Reply:
That's a good suggestion. I think I'll do that.